dev/com.ibm.ws.microprofile.config.1.2/resources/com/ibm/ws/microprofile/config12/resources/Config12.nlsprops

```diff
@@ -32,7 +33,7 @@
 # Emergency Config 1.2 error message
 #-----------------------------------------------------------------------------------------------------------------------------
 
-# A Config API error has occurred: {0}
-temporary.CWMCG9998E=CWMCG9998E: A Config API error has occurred: {0}.
+# The following Config API error has occurred: {0}.
+temporary.CWMCG9998E=CWMCG9998E: The following Config API error has occurred: {0}.
```

Comment:
Are you allowed to change this message?

Reply:
yes ... this change was requested by ID (albeit in the new message file)